frontend/src/hooks/use-app-form/use-app-form.hook.ts

```diff
@@ -1,4 +1,9 @@
-import { useForm, UseFormHandleSubmit } from 'react-hook-form';
+import {
+  useForm,
+  UseFormHandleSubmit,
+  UseFormReset,
+  FieldValues,
+} from 'react-hook-form';
```

Comment:
use type from our types folder

Reply:
Do you want me to transfer types?

Reply:
Only that we don't have
most of them we already have